docs/design/bl.md

```diff
@@ -1,85 +1,625 @@
+hours. Cardano also has probabilistic finality within this window, and in
+practice rollbacks are rarely larger than a few blocks. A block is produced
+roughly every 20-30 seconds.
+[Kraken.com](https://support.kraken.com/hc/en-us/articles/203325283-Cryptocurrency-deposit-processing-times)
+says it waits ~15 minutes, suggesting they wait for 45 blocks before considering
+a transaction safely on-chain.
+
+##### 3.1.2.2 Ouroboros Peras
+
+Cardano is expecting a significant protocol upgrade with Ouroboros Peras, which
+will introduce much faster finality for some block (thus all blocks beneeth
+them) through [stake-based voting](https://peras.cardano-scaling.org/). This
+feature heavily relies on the fact that we treat stake as a source of trust on
+Cardano blockchain.
+
+##### 3.1.2.3 Mempool and Congestion
+
+Of course any congestion can affect the initial settlement time (when the
+transaction is included in a lock) and influence the total time required from
+the submission of the transaction to its finalization (when the transaction can
+not be rolledback). The maximal throughput of both blockchains is currently in
+the same ballpark.
+
+##### 3.1.2.4 Implications
+
+Finality and fees (discussed below) can affect how fast the lightning network
+topology can be changed - how quickly channels can be opened and operational or
+closed. It can also affect how smoothly the liquidity in that topology can be
+adapted to the demands through operations like `splicing` which adds or removes
+funds from the channel. Liquidity distribution across the topology is important
+because it affects if and how big payment can be routed through the network
+without the need for splitting and transferring it using much more involved
+multi path payment.
+
+Faster finality and cheaper transactions means that CL channels can possibly
+utilize L1 with greater flexibility. At the baseline protocol level we probably
+don't have to necessarily compresses every channel L1 operation into a single
+transaction.
+
+We can of course enable more efficient execution strategies based on muti-sig
+transactions and rely as BLN on a full off-chain consensus bewteen partners. As
+we will learn in a moment this seemingly optimal multi-sig route can impose
+inflexibility which can lead to actually more transactions in many contexts!
+
+#### 3.1.3 Lightning Payment Finality
+
+While the discussion above relates to Layer 1 (L1) finality, it's important to
+note that once Lightning Channel is established payments exectued on that layer
+achieve nearly instant payment finality. Successful payment paths ensure that
+transactions are finalized predominantly by network speed.
+
+#### 3.2 Data Availability and Lightweight Clients
+
+##### 3.2.1 BLN "lightweight" clients
+
+Blockchain data availability is crucial for the safety of the BLN channels. The
+main channel UTxO's on-chain presence ensures that Layer 2 operations are secure
```

Comment:
Is this not L2? 

If there is a system already in place, like long hand for first occurence in each section, then just ignore me

Reply:
You are right - I went too far probably with this whole section as it mainly relates to the L2. Can I leave it as we have in the intro that we will somewhat explore that area as well?